js/battledata.js

```diff
@@ -622,6 +622,9 @@ var Tools = {
 			});
 			// [[blah]]
 			str = str.replace(/\[\[(?![< ])(?:(?:(youtube|yt|wiki)\: ?)?([^<`]*?[^< ])?)\]\]/g, function (match, p1, p2) {
+				if (match === '[[]]') {
+					return (p1 ? p1 : '') + match + (p2 ? p2 : '');
```

Comment:
(I didn't really review before merging because this function is being replaced soon)